src/base_decorator.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'Nameable'
```

Comment:
- [ ] Hello! 👋 It's essential to pay attention to the naming conventions for relative path files in your project. To ensure that your project runs smoothly as expected, please make sure that the file names are all in lowercase. This is important for consistency and compatibility with various systems and platforms. 🚀

```suggestion
require_relative 'nameable'
```